docs/source/integrations/lightning_flash.rst

```diff
@@ -0,0 +1,446 @@
+        
+            # 1. Load your FiftyOne dataset
+            dataset = foz.load_zoo_dataset("cifar10", split="test", max_samples=40)
+            train_dataset = dataset.shuffle(seed=51)[:20]
+            test_dataset = dataset.shuffle(seed=51)[20:25]
+            val_dataset = dataset.shuffle(seed=51)[25:30]
+            predict_dataset = dataset.shuffle(seed=51)[30:40]
+        
+            # 2. Load the Datamodule
+            datamodule = ImageClassificationData.from_fiftyone_datasets(
+                train_dataset = train_dataset,
+                test_dataset = test_dataset,
+                val_dataset = val_dataset,
+                predict_dataset = predict_dataset,
+                label_field = "ground_truth",
+                batch_size=4,
+                num_workers=4,
+            )
+        
+            # 3. Build the model
+            model = ImageClassifier(
+                backbone="resnet18", 
+                num_classes=datamodule.num_classes, 
+                serializer=FiftyOneLabels(),
+            )
+        
+            # 4. Create the trainer
+            trainer = Trainer(
+                max_epochs=1, 
+                limit_train_batches=1, 
+                limit_val_batches=1,
+            )
+            
+            # 5. Finetune the model
+            trainer.finetune(
+                model, 
+                datamodule=datamodule,
+                strategy=FreezeUnfreeze(unfreeze_epoch=1),
+            )
+            
+            # 6. Save it!
+            trainer.save_checkpoint("image_classification_model.pt")
+        
+            # 7. Generate predictions
+            model = ImageClassifier.load_from_checkpoint(
+                "https://flash-weights.s3.amazonaws.com/image_classification_model.pt"
+            )
+            model.serializer = FiftyOneLabels()
+
+            predictions = trainer.predict(model, datamodule=datamodule)
+            
+            # 7b. Flatten batched predictions
+            predictions = list(itertools.chain.from_iterable(predictions))
+        
+            # 8. Add predictions to dataset and analyze 
+            predict_dataset.set_values("flash_predictions", predictions)
+            session = fo.launch_app(view=predict_dataset)
+        
+
+    .. tab:: Image object detection 
+
+        This example trains a Flash object detection model on a FiftyOne dataset
+        with |Detections| ground truth labels.
+        
+        .. code-block:: python
+            :linenos:
+
+            import itertools
+            
+            import fiftyone as fo
+            import fiftyone.zoo as foz
+        
+            from flash import Trainer
+            from flash.image import ObjectDetectionData, ObjectDetector
+            from flash.image.detection.serialization import FiftyOneDetectionLabels
+        
+            # 1. Load your FiftyOne dataset
+            dataset = foz.load_zoo_dataset("quickstart", max_samples=40)
```

Comment:
Might be nice to add a comment here that you're using views into one dataset, but using different datasets for each split is also allowed